lib/mm_network.pm

```diff
@@ -7,7 +7,7 @@ use Exporter;
 
 use testapi;
 
-our @EXPORT = qw/configure_hostname get_host_resolv_conf configure_static_ip configure_dhcp configure_default_gateway configure_static_dns parse_network_configuration ip_in_subnet check_ip_in_subnet/;
+our @EXPORT = qw(configure_hostname get_host_resolv_conf configure_static_ip configure_dhcp configure_default_gateway configure_static_dns parse_network_configuration ip_in_subnet check_ip_in_subnet);
```

Comment:
I guess we should sync the perltidy settings as well - the lines are much longer here